l10n_de_website_sale_accept_terms/views/views.xml

```diff
@@ -0,0 +1,312 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+
+        <!-- Add needed CSS and JS in template -->
+        <template id="assets_frontend" inherit_id="website.assets_frontend" name="Accept Terms on Checkout">
+          <xpath expr="." position="inside">
+                <link rel='stylesheet' href='/l10n_de_website_sale_accept_terms/static/src/css/style.css'/>
+                <script type="text/javascript" src="/l10n_de_website_sale_accept_terms/static/src/js/script.js"></script>
+          </xpath>
+        </template>
+
+        <!-- Replace the payment-site -->
```

Comment:
I would recomment to use just extend the payment site 

        <!-- Replace the payment-site -->
        <template id="wsga_payment" name="Checkout with Accept Terms" inherit_id="website_sale.payment">

             <xpath expr="//div[@class='row']" position="after">
                      <div class="wsga-mb30">
                          <h4 id="rev">Terms and revocation!</h4>
                          <div id="revlabel" class="form-group">
                              <p>With click on the button "Pay now" I accept the <span id="open-terms" class="wsga-cursor">terms</span> and the <span id="open-revo" class="wsga-cursor">revocation</span>.</p>
                          </div>
                      </div>
              <div id="wsga-terms" class="wsga-popover wsga-center" style="display:none;">
                    <h3 class="popover-title">
                        Terms
                        <button id="close-terms" title="close" type="button" class="close">X</button>
                    </h3>
                    <iframe src="../../page/website.popover_terms"/>
                </div>


              <div id="wsga-revo" class="wsga-popover wsga-center" style="display:none;">
                    <h3 class="popover-title">
                        Revocation
                        <button id="close-revo" title="close" type="button" class="close">X</button>
                    </h3>
                    <iframe src="../../page/website.popover_revocation/"/>
              </div>

            </xpath>
        </template>